rollup.config.js

```diff
@@ -17,6 +17,17 @@ import pkg from './package.json';
 const getFilteredFolders = files => files.filter(name => name.indexOf('.') === -1);
 
 const getFiles = path => {
+  // eslint-disable-next-line compat/compat
+  // eslint-disable-next-line no-shadow
```

Comment:
We should remove all those comments

Reply:
sorry, my eslint got really crazy and started to add these lines automatically … I need to fix my setup 😉